packages/apps/desktop-viewer-test/src/frontend/components/home/Recents.tsx

```diff
@@ -36,25 +32,11 @@ export const Recents = () => {
         if (displayValue.length > 25) {
           displayValue = `${displayValue.substr(0, 25)}...`;
         }
-        switch (recent.type) {
-          case ViewerFileType.SNAPSHOT:
-            return (
-              <span key={recent.path} onClick={() => openSnapshot(recent.path)}>
-                {displayValue}
-              </span>
-            );
-          case ViewerFileType.ONLINE:
-            return (
-              <span
-                key={recent.iModelId}
-                onClick={() => openRemote(recent.iTwinId, recent.iModelId)}
-              >
-                {displayValue}
-              </span>
-            );
-          default:
-            return null;
-        }
+        return (
+          <span key={recent.path} onClick={() => openFile(recent.path)}>
```

Comment:
i wouldve assumed path is unique, but got a warning in the console about list needed unique keys

Reply:
probably because the tooling cant guarantee uniqueness based on its type......we are only every persisting a single path once so it should be, but I'll look at alternatives